framework/src/engine/consensus/consensus.ts

```diff
@@ -966,6 +966,7 @@ export class Consensus {
 				const { result: verifyResult } = await this._abi.verifyTransaction({
 					contextID,
 					transaction: transaction.toObject(),
+					header: block.header.toObject(),
```

Comment:
I thought we wanted to pass partial header object,
```ts
{
  height: number;
  timestamp: number
}
```

Reply:
https://github.com/LiskHQ/lisk-sdk/pull/7606/files#diff-0bac9c6e42245fd8bdb1b7378788978dc188e9c225a0d91c5b5f08e28dab2e9bR76

here it's passing partial block header to the module. For the ABI level, it has `context` as well, so it will be inconsistent to pass only those 2 properties.
Probably it's better to revisit the ABI schema, ill create an issue for it

Reply:
yeah we can think about it when we revisit ABI schema 👍